VultisigApp/VultisigApp/macOS/View/VultisigApp+macOS.swift

```diff
@@ -9,36 +9,24 @@
 import SwiftUI
 
 extension VultisigApp {
-    var content: some Scene {
-        WindowGroup {
-            ContentView()
-                .environmentObject(applicationState) // Shared monolithic mutable state
-                .environmentObject(vaultDetailViewModel)
-                .environmentObject(coinSelectionViewModel)
-                .environmentObject(accountViewModel)
-                .environmentObject(deeplinkViewModel)
-                .environmentObject(settingsViewModel)
-                .environmentObject(homeViewModel)
-                .environmentObject(settingsDefaultChainViewModel)
-                .environmentObject(macCheckUpdateViewModel)
-                .environmentObject(macCameraServiceViewModel)
-                .onChange(of: scenePhase) {
-                    switch scenePhase {
-                    case .active:
-                        continueLogin()
-                    case .background:
-                        resetLogin()
-                    default:
-                        break
-                    }
-                }
-                .buttonStyle(BorderlessButtonStyle())
-                .frame(minWidth: 900, minHeight: 600)
-                .onAppear{
-                    NSWindow.allowsAutomaticWindowTabbing = false
-                }
-        }
-        .modelContainer(sharedModelContainer)
+    var content: some View {
+        ContentView()
+            .environmentObject(applicationState) // Shared monolithic mutable state
+            .environmentObject(vaultDetailViewModel)
+            .environmentObject(coinSelectionViewModel)
+            .environmentObject(accountViewModel)
+            .environmentObject(deeplinkViewModel)
+            .environmentObject(settingsViewModel)
+            .environmentObject(homeViewModel)
+            .environmentObject(settingsDefaultChainViewModel)
+            .environmentObject(vultExtensionViewModel)
+            .environmentObject(macCheckUpdateViewModel)
+            .environmentObject(macCameraServiceViewModel)
+            .buttonStyle(BorderlessButtonStyle())
+            .frame(minWidth: 900, minHeight: 600)
+            .onAppear{
+                NSWindow.allowsAutomaticWindowTabbing = false
+            }
```

Comment:
so we don't need to monitor `scenePhase`?

Reply:
We do but only for iOS. We use that observer to lock the app when user dismisses the app. On Mac, user can have multiple apps opened at once.